test/tests/interfacekernels/InterfaceSorption/interface_sorption.i

```diff
@@ -20,8 +26,7 @@ R = 8.31446261815324 # Based on PhysicalConstants
   [gen]
     type = GeneratedMeshGenerator
     nx = 10
-    ny = 10
-    dim = 2
+    dim = 1
```

Comment:
Because we really do not need the 2D. It doesn't test anything more since this is a fundamentally 1D problem. 
So I'm just reducing the cost of tests without loss in testing and accuracy.

Reply:
Sounds good!